app/main.py

```diff
@@ -1,8 +1,55 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        price = 0.0
+        if car.clean_mark < self.clean_power:
+            points_to_clean = self.clean_power - car.clean_mark
+            price_formula_up = car.comfort_class * \
+                points_to_clean * self.average_rating
+            price += price_formula_up / self.distance_from_city_center
```

Comment:
Reformat the code without creating new variables and omit using line breaks `\`, prefer using `()`, and place your calculations there. It will make it possible to assess the expressions at first glance and will not allocate additional memory for storing intermittent results.